pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -30,11 +30,11 @@
                   'auto_resolve',
                   'ignore_base',
                   'final_repo_dir']
-_PYTHON_PUB_REQUIRED = ['pypi_server_url',
-                        'pypi_uname',
-                        'pypi_pwd',
-                        'publish_env']
+_PUB_REQUIRED = ['server_url',
+                 'user_name',
+                 'password']
 _PYTHON_PUB_ENVS = ['dev', 'test', 'staging', 'prod']
+_JAVA_PUB_ENVS = ['prod']
```

Comment:
Since the pipeline currently doesn't support snapshot release, should include staging?